### PR TITLE
[zh-cn] Localize blog 2022-09-12-Announcing-the-Auto-Refreshing-Offic…

### PR DESCRIPTION
**Pull request by jzhupup**
Added the zh-cn text to this blog:
```
content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md
````

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+(also called "CVEs", after the database that tracks public security issues across
+different products and vendors). Accompanying the release of Kubernetes v1.25,
+we are excited to announce availability of such
+a [feed](/docs/reference/issues-security/official-cve-feed/) as an `alpha`
+feature. This blog will cover the background and scope of this new service. 
+-->
+**作者**：Pushkar Joglekar (VMware)
+
+Kubernetes 社区的一个长期存在的需求是为最终用户提供一种可编程方式来跟踪
+Kubernetes 安全问题（也称为“CVE”，来自于跟踪不同产品和供应商的公共安全问题的数据库）。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:28]:
```suggestion
Kubernetes 社区有一个长时间未解决的需求，即为最终用户提供一种编程方式来跟踪
Kubernetes 安全问题（也称为“CVE”，这来自于跟踪不同产品和供应商的公共安全问题的数据库）。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Kubernetes have increased. Although most CVEs that directly, indirectly, or
+transitively impact Kubernetes are regularly fixed, there is no single place for
+the end users of Kubernetes to programmatically subscribe or pull the data of
+fixed CVEs. Current options are either broken or incomplete. 
+-->
+## 动机
+
+随着关注 Kubernetes 的人越来越多，与 Kubernetes 相关的 CVE 数量也在增加。
+尽管大多数直接地、间接地或传递性地影响 Kubernetes 的 CVE 都被定期修复，
+Kubernetes 的最终用户没有一个地方可以以编程方式来订阅或拉取固定的 CVE 数据。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:46]:
```suggestion
但 Kubernetes 的最终用户没有一个地方可以以编程方式来订阅或拉取固定的 CVE 数据。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+目前的选项已损坏或是不完整的。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:47]:
```suggestion
目前的一些数据源要么已损坏，要么不完整。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+<!-- 
+## What's Next?
+
+In order to graduate this feature, SIG Security
+is gathering feedback from end users who are using this alpha feed. 
+-->
+## 下一步工作
+
+为了完善此功能，SIG Security 正在收集使用此 alpha 订阅源的最终用户的反馈。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:104]:
```suggestion
为了完善此功能，SIG Security 正在收集使用此 Alpha 订阅源的最终用户的反馈。
```

**Review by windsonsea — COMMENTED** [review]
To improve readability

**Comment by windsonsea** [comment]
The en upstream is updated. See:
https://github.com/kubernetes/website/pull/36292/files

**Comment by jzhupup** [comment]
> The en upstream is updated, so the localized text need be revised too. See: https://github.com/kubernetes/website/pull/36292/files

done

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+* **Platform Providers**: Persons or teams who _manage_ Kubernetes clusters
+* **Maintainers**: Persons or teams who _create_ and _support_ Kubernetes
+  releases through their work in Kubernetes Community - via various Special
+  Interest Groups and Committees. 
+  -->
+### 针对的受众
+
+* **最终用户**：**使用** Kubernetes 部署他们的应用程序的个人或团队。
+* **平台提供商**：**管理** Kubernetes 集群的个人或团队。
+* **维护人员**：通过各种特殊兴趣小组和委员会在 Kubernetes 社区中**创建**和**支持** Kubernetes
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:93]:
```suggestion
* **维护人员**：通过各种特别兴趣小组和委员会在 Kubernetes 社区中**创建**和**支持** Kubernetes
```

**Review by windsonsea — COMMENTED** [review]
Some more that you can find if you read more.

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+transitively impact Kubernetes are regularly fixed, there is no single place for
+the end users of Kubernetes to programmatically subscribe or pull the data of
+fixed CVEs. Current options are either broken or incomplete. 
+-->
+## 动机
+
+随着关注 Kubernetes 的人越来越多，与 Kubernetes 相关的 CVE 数量也在增加。
+尽管大多数直接地、间接地或传递性地影响 Kubernetes 的 CVE 都被定期修复，
+但 Kubernetes 的最终用户没有一个地方能够以编程方式来订阅或拉取固定的 CVE 数据。
+目前的一些数据源要么已损坏，要么不完整。。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:47]:
```suggestion
目前的一些数据源要么已损坏，要么不完整。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+So in order to improve the feed in future Kubernetes Releases, if you have any
+feedback, please let us know by adding a comment to
+this [tracking issue](https://github.com/kubernetes/sig-security/issues/1) or
+let us know on
+[#sig-security-tooling](https://kubernetes.slack.com/archives/C01CUSVMHPY)
+Kubernetes Slack channel.
+(Join [Kubernetes Slack here](https://slack.k8s.io)) 
+-->
+因此，为了在未来的 Kubernetes 版本中改进订阅源，如果你有任何反馈，请通过添加评论至
+[tracking issue](https://github.com/kubernetes/sig-security/issues/1) 告诉我们，
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:131]:
```suggestion
[問題追蹤](https://github.com/kubernetes/sig-security/issues/1)告诉我们，
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+* Triage and vulnerability disclosure will continue to be done by SRC (Security
+  Response Committee).
+* Listing CVEs that are identified in build time dependencies and container
+  images are out of scope.
+* Only official CVEs announced by the Kubernetes SRC will be published in the
+  feed. 
+-->
+### 不能做什么
+
+* 脆弱性的分类和披露将继续由 SRC（Security Response Committee，安全响应委员会）完成。
```
[the remark this answers is not in the document]
Comment (jzhupup) [reply content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:75]: This translation is tengqm's suggestion

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+(also called "CVEs", after the database that tracks public security issues across
+different products and vendors). Accompanying the release of Kubernetes v1.25,
+we are excited to announce availability of such
+a [feed](/docs/reference/issues-security/official-cve-feed/) as an `alpha`
+feature. This blog will cover the background and scope of this new service. 
+-->
+**作者**：Pushkar Joglekar (VMware)
+
+Kubernetes 社区有一个长时间未解决的需求，即为最终用户提供一种编程方式来跟踪
+Kubernetes 安全问题（也称为“CVE”，这来自于跟踪不同产品和供应商的公共安全问题的数据库）。
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:28]:
```suggestion
Kubernetes 安全问题（也称为 “CVE”，这来自于跟踪不同产品和供应商的公共安全问题的数据库）。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+随着 Kubernetes v1.25 的发布，我们很高兴地宣布以 `alpha`
+特性的形式推出这样的[订阅源](/zh-cn/docs/reference/issues-security/official-cve-feed/)。
+本博客将介绍这项新服务的背景和作用域。
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:31]:
```suggestion
本文将介绍这项新服务的背景和作用域。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+  Response Committee).
+* Listing CVEs that are identified in build time dependencies and container
+  images are out of scope.
+* Only official CVEs announced by the Kubernetes SRC will be published in the
+  feed. 
+-->
+### 不能做什么
+
+* 脆弱性的分类和披露将继续由 SRC（Security Response Committee，安全响应委员会）完成。
+* 列出在构建时依赖项和容器镜像中发现的 CVE 不在范围之内。
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:76]:
```suggestion
* 不会列出在构建时依赖项和容器镜像中发现的 CVE。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md (hunk 10 above)
Comment (jzhupup) [reply content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md]: I have revised the others

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**